nginx/Dockerfile

```diff
@@ -1,3 +1,6 @@
 FROM nginx
 
-COPY nginx.conf /etc/nginx/nginx.conf
+MAINTAINER Mark Robinson <robinm8@rpi.edu>
+
+COPY conf/nginx.conf /etc/nginx/nginx.conf
+COPY ssl/ /etc/nginx/ssl/
```

Comment:
ssl/ required

For security reasons, docker can not break out of build working directory.

The ssl folder has to be inside the build directory of nginx. When I moved the ssl folder to the parent directory, docker could not see the ssl folder because it was not in the nginx build context.